docs/pages/zero-trust-access/zero-trust-access.mdx

```diff
@@ -1,6 +1,286 @@
+      href: "./sso/",
+      tags: [
+        {
+          name: "Okta",
+          href: "./sso/okta/",
+          icon: "okta",
+        },
+        {
+          name: "GitHub",
+          href: "./sso/github-sso/",
+          icon: "github",
+        },
+        {
+          name: "Entra ID",
+          href: "./sso/entra-id-oidc/",
+          icon: "entraId",
+        },
+        {
+          name: "G Suite",
+          href: "./sso/google-workspace/",
+          icon: "googleCloud",
+        },
+       {
+          name: "more",
+          href: "./sso/",
+          arrow: true,
+        },
+      ]
+    },  
+    {
+      title: "Manage and Track Teleport Audit Events",
+      description: "View audit log generated when users and services interact with your cluster. And export events to store them in a log management platform.",
+      href: "./export-audit-events/",
+      tags: [
+        {
+          name: "Elastic Stack",
+          href: "./export-audit-events/elastic-stack/",
+          icon: "elasticStack",
+        },
+        {
+          name: "Panther",
+          href: "./export-audit-events/panther/",
+          icon: "panther",
+        },
+        {
+          name: "Datadog",
+          href: "./export-audit-events/datadog/",
+          icon: "datadog",
+        },
+        {
+          name: "Splunk",
+          href: "./export-audit-events/splunk/",
+        },
+      ]
+    },
+    {
+      title: "Self-Hosting Teleport",
+      description: "Run Teleport Enterprise in your own infrastructure from setting up high availability and multi-region clusters to secure CA keys with KMS or HSM.",
+      href: "./deploy-a-cluster/",
+      tags: [
+        {
+          name: "AWS KMS",
+          href: "./deploy-a-cluster/aws-kms/",
+          icon: "aws",
+        },
+        {
+          name: "Google Cloud KMS",
+          href: "./deploy-a-cluster/gcp-kms/",
+          icon: "gSuite",
+        },
+        {
+          name: "Helm Chart",
+          href: "./deploy-a-cluster/helm-deployments/",
+        },
+      ]
+    },  
+    {
+      title: "Infrastructure as Code",
+      description: "Manage Teleport's Dynamic Resources using infrastructure as code tools, including Terraform, Helm and the Teleport tctl client tool.",
+      href: "./infrastructure-as-code/",
+      tags: [
+        {
+          name: "Terraform",
+          href: "./infrastructure-as-code/terraform-provider/",
+          icon: "terraform",
+        },
+        {
+          name: "Kubernetes Operator",
+          href: "./infrastructure-as-code/teleport-operator/",
+          icon: "kubernetes2",
+        },
+        {
+          name: "Helm Chart",
+          href: "./deploy-a-cluster/helm-deployments/",
+        },
+      ]
+    },
+    {
+      title: "Security Best Practices",
+      description: "To reduce system vulnerabilities and to avoid security follow recommended security practices to secure your cluster.",
```

Comment:
The descriptions for the other topics always starts with a verb. How about this instead?


```suggestion
      description: "Secure your cluster with these best practices.",
```

If you don't like the suggested wording, that's okay too, but we do need to fix this either way as "avoid security follow" doesn't read correctly.